source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -66,22 +66,18 @@
 CO2_ESTIMATION_COEFF_1 = 1.35
 CO2_ESTIMATION_COEFF_2 = 184
 
+
 # Possible values implemented. Full languages within the doc:
 # https://developer.here.com/documentation/routing/dev_guide/topics/resource-param-type-languages.html#languages
 # Be careful, the syntax has to be exact
 class Languages(Enum):
-    afrikaans = "af"
-    arabic = "ar-sa"
-    chinese = "zh-cn"
     dutch = "nl-nl"
     english = "en-gb"
```

Comment:
I made a comment related to a default value being `en-US` instead of `en-GB` for Asgard. If you decide to change the default value in Asgard, you probably need to modify this one too for coherence.

Reply:
:+1: